.idea/dictionaries/Ben.xml

```diff
@@ -0,0 +1,8 @@
+<component name="ProjectDictionaryState">
+  <dictionary name="Ben">
+    <words>
+      <w>chillabit</w>
```

Comment:
You added this as well in one of your branches. It's a dictionary addition so that "refactoring" doesn't show this as something that needs to be refactored.